modules/extension/wmts/src/main/java/org/geotools/ows/wmts/WebMapTileServer.java

```diff
@@ -205,46 +212,142 @@ public GetTileResponse issueRequest(GetSingleTileRequest tileRequest)
         return (GetTileResponse) internalIssueRequest(tileRequest);
     }
 
-    /** Creates a GetMultiTileRequest, signature kept for now. */
+    /** Creates a GetMultiTileRequest. */
     public GetTileRequest createGetTileRequest() {
 
-        URL url;
         OperationType getTile = getCapabilities().getRequest().getGetTile();
-        if (getTile == null) {
```

Comment:
what happens now if `getTile` returns null because there is no getTile element in the capabilities?

Reply:
@ianturton Good question. The function findGetTileURL will return null, and then we will have the same setting as before.
I would very much like to add unit tests to cover all possible configurations.

Reply:
I think `testGEOT6741` should cover this situation, so I guess we're good.